deltachat-ios/ViewModel/ChatListViewModel.swift

```diff
@@ -342,7 +355,6 @@ extension ChatListViewModel: UISearchResultsUpdating {
         } else {
             inBgSearch = true
             DispatchQueue.global(qos: .userInteractive).async { [weak self] in
-                usleep(100000)
```

Comment:
not sure if you have tested that with a 3gb database :)

on larger databases, a search easily takes some 100ms - this initital delay is to let users type some more characters, resulting in overall faster and responsive results.

so, i would leave that out of this pr, i think, it is still fast enough.

Reply:
ok, then I'll undo this change

Reply:
Typing multiple letters in 100ms is only sth. for quite advanced users. I wonder if we should increase the initial delay up to 250 ms.  However this might be an option for another PR and can be discussed separately.